framework/test/unit/engine/generator/network_endpoint.spec.ts

```diff
@@ -203,12 +203,51 @@ describe('generator network endpoint', () => {
 				expect(pool.add).toHaveBeenCalledTimes(2);
 			});
 
-			it('should apply penalty when validateTransactions fails', async () => {
+			it('should not apply penalty when abi verification fails', async () => {
```

Comment:
test description seems to be ok, but from the source code, i don't understand how it will not call the applyPenalty 🤔

Reply:
Updated the tests.